README.md

```
@@ -306,16 +306,16 @@ just lint              # Check ruff + black formatting
+just repo              # Build release + generate Kodi repo in dist/
 just repo-zip          # Build repo + copy repository zip to cwd
 just clean             # Remove build artifacts
-just dist-clean        # Remove build artifacts + repo/zips/
+just dist-clean        # Remove build artifacts + dist/
 ```
 
 ### Project Structure
 
-```text
-repo/plugin.video.nzbdav/
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)